### PR TITLE
Basic electron app setup

### PR DESCRIPTION
**Pull request by krzysztof-jelski**
Basics of Electron app setup, that allows to run current version of application as an Electron app.

Looking for feedback especially with regards to using polkadot dev packages and scripts, package naming and typescript setup.

### CODE REVIEW

File: packages/apps/src/index.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ import settings from '@polkadot/ui-settings';
 
 import Apps from './Apps';
 
+initSettings();
```
Comment (jacogr) [inline packages/apps/src/index.tsx:22]:
```suggestion
initSettings();

```

File: packages/apps/src/initSettings.ts [hunk 2]
```diff
@@ -8,36 +8,38 @@ import { createEndpoints } from '@polkadot/apps-config/settings';
 import { registry } from '@polkadot/react-api';
 import settings from '@polkadot/ui-settings';
 
+export default function (): void {
```
Comment (jacogr) [inline packages/apps/src/initSettings.ts:11]:
```suggestion
export default function initSettings (): void {
```

File: packages/electron-app/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@polkadot/electron-app",
```
Comment (jacogr) [inline packages/electron-app/package.json:2]: Would probably name this `apps-electron` (Just organizes the `@polkadot/*` namespace a slightly better)

File: packages/electron-app/src/index.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import { ThemeProvider } from 'styled-components';
+import settings from '@polkadot/ui-settings';
+import Queue from '@polkadot/react-components/Status/Queue';
+import { BlockAuthors, Events } from '@polkadot/react-query';
+import AccountSidebar from '@polkadot/app-accounts/Sidebar';
+import { Api } from '@polkadot/react-api';
+import Apps from '@polkadot/apps/Apps';
+import initSettings from '@polkadot/apps/initSettings';
+
+initSettings();
```
Comment (jacogr) [inline packages/electron-app/src/index.tsx:22]:
```suggestion
initSettings();

```

File: packages/electron-app/src/index.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                <HashRouter>
+                  <Apps />
+                </HashRouter>
+              </AccountSidebar>
+            </Events>
+          </BlockAuthors>
+        </Api>
+      </Queue>
+    </ThemeProvider>
+  </Suspense>,
```
Comment (jacogr) [inline packages/electron-app/src/index.tsx:50]: Really wondering if we can't expose this block as a function with `settings` passed in inside apps. I will forget to update 2 places...

File: packages/electron-app/webpack.main.config.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+                loader: require.resolve('babel-loader'),
+                options: require('@polkadot/dev/config/babel')
+              }
+            ]
+          }
+        ]
+      },
+      optimization: {
+        minimize: !!isProd,
+        minimizer: [new TerserPlugin()]
```
Comment (jacogr) [inline packages/electron-app/webpack.main.config.js:34]: Happy to give this a go.

File: packages/electron-app/webpack.renderer.config.js [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const findPackages = require('../../scripts/findPackages');
+
+const ENV = process.env.NODE_ENV || 'development';
+
+function createWebpack ({ alias = {}, context, name = 'index' }) {
```
Comment (jacogr) [inline packages/electron-app/webpack.renderer.config.js:18]: Would prefer to expose the common stuff as "common" and include it in both configs. Easier to maintain.

**Comment by krzysztof-jelski** [comment]
Closing, as it's not possible to change the base branch. New PR will be from this repo's branch.

File: packages/apps/src/initSettings.ts (hunk 2 above)
Comment (Jozwiaczek) [reply packages/apps/src/initSettings.ts:11]: done

File: packages/electron-app/package.json (hunk 3 above)
Comment (Jozwiaczek) [reply packages/electron-app/package.json:2]: done

**Pull request closed without merging**